pkg/config/load.go

```diff
@@ -33,6 +36,24 @@ type Prowgen struct {
 	Expose bool `json:"expose,omitempty"`
 	// Rehearsals declares any disabled rehearsals for jobs
 	Rehearsals Rehearsals `json:"rehearsals,omitempty"`
+	// Set which architecture should the images be promoted from
+	AdditionalArchitectures []api.Architecture `json:"additional_architectures"`
+}
+
+func (p *Prowgen) Validate() error {
+	errs := make([]error, 0)
+	invalidArchs := make([]string, 0, len(p.AdditionalArchitectures))
```

Comment:
Shall we care to display which archs are invalid? If, so, you can just do it with sets and intersections.